Elastica/EventSubscriber/UpdateContentIndexedSubscriber.php

```diff
@@ -33,14 +34,12 @@ public function updateIndexedContent(ContentEvent $event)
     {
         $content = $event->getContent();
 
-        if (!$content->getStatus()->isPublished()) {
-            return;
-        }
-
         $lastPublishedContent = $this->contentRepository->findLastPublishedVersion($content->getContentId(), $content->getLanguage());
 
-        if ($content->getVersion() >= $lastPublishedContent->getVersion()) {
-            $this->contentIndexor->index($content);
+        if ($lastPublishedContent instanceof ContentInterface) {
```

Comment:
@itkg-ngilain, I think that the cases you listed are not full : 

- what do I do if the Content before the content included in the event is still publish ?
- what do I do if the content in the event is published but is not the last published version of the content ?